modules/ROOT/pages/glossary.adoc

```diff
@@ -15,69 +15,72 @@ endif::[]
 [[secta]]
 *A*
 
-Activity:: Role-based document transaction tracking (for send and receive) based on rules, actions, and functions.
+Acknowledgment:: Sent as a response to a B2B transmission. A TA1 acknowledgment serves as a receipt that a transmission was received. A functional acknowledgment includes details of the validation process.
+
+Activity:: A list of transmissions sent to and received from partners.
+
+Activity API:: Provides users with the ability to obtain transaction status.
+
+Anypoint Partner Manager:: Configures and manages integrations between EDI transmissions with your trading partners and your backend APIs; also tracks and reports on the transmission activity.
 
-Anypoint Partner Manager:: A single web-based user interface for Partner Configuration, Document flow  and Transaction Monitoring. The functionality that serves as a translation layer between internal systems and the various formats (X12, EDIFACT, XML, CSV, and others) and transport protocols (AS2, FTP/S, and others) commonly used in electronic transactions between companies.
 
-Activity API:: Provides users with the ability to obtain transaction status???
 
 [[secte]]
 *E*
 
-Endpoint:: Defines the protocol, address, and other details specific to an exchange of messages between partners such as Acknowledgement, Source Endpoint, and Target Endpoint.
+Endpoint: Source at host:: The origin of the message flow when defined by a host endpoint and message type, often an API.
```

Comment:
You're using "endpoint" to define itself. How about, "The origin of a message, defined by its host and message type. An endpoint is typically a URL to an API."  -- APIs may be an endpoint, but you need a URL to access them.

Reply:
We will probably not use any definition for endpoint by itself. We have defined source and target endpoints in the docs and that should be sufficant.